cmd/crio/main.go

```diff
@@ -99,6 +100,28 @@ func validateConfig(config *server.Config) error {
 		}
 	}
 
+	for _, d := range config.RuntimeConfig.AdditionalDevices {
+		split := strings.Split(d, ":")
+		switch len(split) {
+		case 3:
+			if !createconfig.IsValidDeviceMode(split[2]) {
+				return fmt.Errorf("invalid device mode: %s", split[2])
+			}
+			fallthrough
+		case 2:
+			if !createconfig.IsValidDeviceMode(split[1]) && !strings.HasPrefix(split[1], "/dev/") {
```

Comment:
do we need to ensure that `len(split)==2` (and not 3) when it is a valid device mode here?

Reply:
maybe not? under case 2,  
if `len(split) == 2`,  it's valid if `split[1]`is permissions or /dev/.. , 
if `len(split) == 3`, it's valid if `split[1]` is /dev/.. 
so this case `/dev:rmw:rmw` hasn't been checked, but podman doesn't check this, just uses split[1] as permission.

Reply:
Correct 
/dev/foo
/dev/foo:rwm
/dev/foo:/dev/bar:rwm 

Are all valid, I believe

Reply:
yes my observation was about catching `/dev:rmw:rmw` as invalid